packages/office-ui-fabric-react/src/components/Breadcrumb/__snapshots__/Breadcrumb.test.tsx.snap

```diff
@@ -136,16 +136,6 @@ exports[`Breadcrumb renders breadcumb correctly 1`] = `
                 TestText4
               </div>
             </span>
-            <i
-              aria-hidden={true}
```

Comment:
This seems concerning. What is the icon missing in the snapshot?

Reply:
Also can you please add a separate simple snapshot test that renders the breadcrumb with a different icon?